### DISCUSSION TITLE
Option to make disabled / readonly conditional

### DISCUSSION THREAD

**Discussion by adamberecz** [vueform/vueform#112] [Ideas]
Option to make disabled / readonly conditional

**Comment by nclemeur** [comment]
Just wondering, if this could be done as part of a plugin while waiting for the official implementation? Currently I am working around this limitation by having two elements for a single field, one for it's disabled state and one for the enabled state and conditions that are inverse of each other... That's not really ideal.

**Reply by nclemeur** [reply]
Just coming back to this, I am really thinking this should be added as it is a very common feature to disable fields depending on the values of other fields. I think that it would be great to leverage all the conditional rendering. For example we could have another prop `conditionalRenderingMode` that would allow to switch from using the visibility (current mode) to disabled or readonly state. It would then be a matter to change how the disabled/readonly/visible values are calculated based on the mode and the conditions...

**Comment by adamberecz** [comment]
Released in `1.10.4` 🥳

https://vueform.com/reference/text-element#option-readonly
https://vueform.com/reference/text-element#option-disabled
